src/Symfony/Component/Form/DependencyInjection/FormPass.php

```diff
@@ -46,29 +50,36 @@ public function process(ContainerBuilder $container)
         }
 
         $definition = $container->getDefinition($this->formExtensionService);
```

Comment:
@HeahDude what do you recommend in case it is not? Throwing or just returning? I'll fix it